src/code/z_lights.c

```diff
@@ -261,12 +261,12 @@ void LightContext_SetAmbientColor(LightContext* lightCtx, u8 r, u8 g, u8 b) {
     lightCtx->ambient.b = b;
 }
 
-void func_80102544(LightContext* lightCtx, u8 a1, u8 a2, u8 a3, s16 numLights, s16 sp16) {
-    lightCtx->unk7 = a1;
-    lightCtx->unk8 = a2;
-    lightCtx->unk9 = a3;
-    lightCtx->unkA = numLights;
-    lightCtx->unkC = sp16;
+void LightContext_SetFogProperties(LightContext* lightCtx, u8 r, u8 g, u8 b, s16 near, s16 far) {
+    lightCtx->fogColor.r = r;
```

Comment:
oot has `LightContext_SetFog` and I think this is fine to use here too. its implied that you are changing properties of the thing